common/lib/capa/capa/responsetypes.py

```diff
@@ -1958,7 +1980,11 @@ def validate_answer(self, answer):
             return False
 
     def get_answers(self):
-        return {self.answer_id: self.correct_answer}
+        _ = self.capa_system.i18n.ugettext
+        # Translators: Separator used in NumericalResponse to display multiple answers.
+        # Example: "Answer: Answer_1 or Answer_2 or Answer_3".
+        separator = u' <b>{}</b> '.format(_('or'))
```

Comment:
Also, this will look like `Answer_1 orAnswer_2` is that expected ?  i don't think so.